docs/transparent-validation/ec-information/ec-explained.md

```diff
@@ -0,0 +1,21 @@
+# EC power states in NovaCustom laptops
```

Comment:
let's make this a bit more generic, I suspect we'll be adding more sections here sooner or later